cms/static/sass/elements/_header.scss

```diff
@@ -388,7 +388,8 @@ body.course.view-export-git .nav-course-tools-export-git,
 body.course.view-team .nav-library-settings .title,
 body.course.view-team .nav-library-settings-team,
 body.course.view-checklists .nav-course-tools .title,
-body.course.view-checklists .nav-course-tools-checklists {
+body.course.view-checklists .nav-course-tools-checklists,
+.nav-course-tools-optimizer {
```

Comment:
The Course Optimizer option is showing correctly but doesn't redirect to the course-optimizer page. The pointer-events: none on the <a> tag is preventing the link from being clickable, and the hover effect is also not working (color doesn’t change to blue).

https://github.com/user-attachments/assets/e13738a8-5239-4188-a146-19e090829507

Reply:
There was some issue with compiling assets. After re-compiling the assets again, it is working perfectly.